src/components/tree-detail/last-waterings/last-waterings.tsx

```diff
@@ -71,12 +71,20 @@ export const LastWaterings: React.FC<LastWateringsProps> = ({
 					<WateringSection
 						waterings={wateringsThisMonth}
 						title={i18n.treeDetail.lastWaterings.thisMonth}
-						noWateringsHint={i18n.treeDetail.lastWaterings.nothingThisMonth}
+						noWateringsHint={
+							wateringsThisWeek.length > 0
+								? i18n.treeDetail.lastWaterings.nothingMoreThisMonth
+								: i18n.treeDetail.lastWaterings.nothingThisMonth
+						}
 					/>
 					<WateringSection
 						waterings={wateringsThisYear}
 						title={i18n.treeDetail.lastWaterings.thisYear}
-						noWateringsHint={i18n.treeDetail.lastWaterings.nothingThisYear}
+						noWateringsHint={
+							wateringsThisWeek.length > 0
```

Comment:
this would not cover the situation when you have one or multiple `wateringsThisMonth`, but 0 `wateringsThisWeek`.